Bootloaders/DFU/makefile

```diff
@@ -11,6 +11,7 @@
 
 # Run "make help" for target help.
 
+LTO			 = Y
```

Comment:
Could this be moved to below this block?

Reply:
Done :)